agents/craftassist/requirements.txt

```diff
@@ -1,2 +1,2 @@
 -r ../../requirements.txt
-eventlet
```

Comment:
why remove `eventlet`?

Reply:
Sorry, it was a mistake, let me change it back. 

> why remove `eventlet`?